elastic/integrations
# [cef] Add Pre-processor config option to CEF manifest

## Proposed commit message

Add a config option to add "pre-processors" which will run before the "decode_cef" processor on the agent. CEF messages from some sources may be malformed by not following the CEF spec exactly, which can then cause problems in the "decode_cef" processor. Having a pre-processor will allow running scripts that can correct the malformed CEF messages, and allow decode_cef to process them correctly.


## Checklist


## How to test this PR locally

Add a processor to the "pre-processor" config option. For example, a script processor that modifies the CEF message by correcting improper escape sequences.  Install the integration with a log input, and feed the logfile incorrect CEF messages. Verify the pre-processor runs and corrects the messages,and decode_cef has no errors
